When I set flag only --enable-cuda-mock, It's not installed.

**Describe the bug**
A clear and concise description of what the bug is.
문제가 발생하는 상황에 대한 요약 설명을 적어주세요.

When I try to install Backend.AI with "--enable-cuda-mock" and "--editable-webui", It is not installed and shows the error message.

```console
➜  backend.ai git:(main) ✗ ./scripts/install-dev.sh  --enable-cuda-mock  --editable-webui

Backend.AI one-line installer for developers

[INFO] Checking prerequisites and script dependencies...
Already up-to-date.
Warning: jq 1.6 is already installed and up-to-date.
To reinstall 1.6, run:
  brew reinstall jq
Detected Docker installation: System package (20.10.17)
Detected docker-compose installation (2.6.1)
validating Docker Desktop mount permissions: ok

[ERROR] You can't use both CUDA and CUDA mock plugins at once!

[ERROR] Please remove --enable-cuda or --enable-cuda-mock flag to continue.
➜  backend.ai git:(main) ✗
```



**To Reproduce**
Steps to reproduce the behavior:
문제를 재현하기 위한 순서를 자세히 적어주세요:
1. try to install Backend.Ai using install-dev.sh  with flags "--enable-cuda-mock" and "--editable-webui"
2. See error
```console
➜  backend.ai git:(main) ✗ ./scripts/install-dev.sh  --enable-cuda-mock  --editable-webui

Backend.AI one-line installer for developers

[INFO] Checking prerequisites and script dependencies...
Already up-to-date.
Warning: jq 1.6 is already installed and up-to-date.
To reinstall 1.6, run:
  brew reinstall jq
Detected Docker installation: System package (20.10.17)
Detected docker-compose installation (2.6.1)
validating Docker Desktop mount permissions: ok

[ERROR] You can't use both CUDA and CUDA mock plugins at once!

[ERROR] Please remove --enable-cuda or --enable-cuda-mock flag to continue.
➜  backend.ai git:(main) ✗
```

**Expected behavior**
A clear and concise description of what you expected to happen.
원래 기대했던 동작은 어떤 것인지 명확하게 적어주세요.

Backend.AI is successfully installed. 

**Screenshots**
If applicable, add screenshots to help explain your problem.
Just take a screenshot in your clipboard or copy a picture file and paste it here (Ctrl+V/Cmd+V) so that GitHub automatically attach and insert it.
가능하다면, 문제 상황을 보여주는 스크린샷을 첨부해주세요.
클립보드에 스크린샷을 찍거나 이미지 파일을 복사해서 여기에 붙여넣으면(Ctrl+V/Cmd+V) GitHub이 자동으로 이곳에 첨부 및 삽입해줍니다.
<img width="1652" alt="image" src="https://user-images.githubusercontent.com/20002/180590949-87c89589-0ed6-4bf9-a66c-7bb3c7f53723.png">



**Server:**
 - OS: macOS 11.4
   - Output of `uname -a`: `Darwin seongdaehyeon-ui-MacBookPro.local 21.5.0 Darwin Kernel Version 21.5.0: Tue Apr 26 21:08:37 PDT 2022; root:xnu-8020.121.3~4/RELEASE_ARM64_T6000 arm64`
 - Backend.AI version: main
 - Python version:  3.10.5
 - Installation method: [used scripts/install-dev.sh]

**Additional context**
Add any other context about the problem here.
추가적으로 문제 상황을 설명할 수 있는 다른 맥락을 기술해주세요.

--- a/changes/578.fix.md
+++ b/changes/578.fix.md
@@ -1,1 +1,1 @@
-fix: checking "--enable-cuda" & "--enable-cuda-mock", modify cp cuda-mock.toml file
+install-dev.sh: Fix "AND" operator when checking `--enable-cuda` &amp; `--enable-cuda-mock` and modify the default-installed `cuda-mock.toml` file

--- a/changes/578.fix.md
+++ b/changes/578.fix.md
@@ -1,0 +1,1 @@
+fix: checking "--enable-cuda" & "--enable-cuda-mock", modify cp cuda-mock.toml file

--- a/scripts/install-dev.sh
+++ b/scripts/install-dev.sh
@@ -562,7 +562,7 @@ if [ "$DISTRO" = "Darwin" ]; then
   echo "${REWRITELN}validating Docker Desktop mount permissions: ok"
 fi
 
-if [ $ENABLE_CUDA -eq 1 ] & [ $ENABLE_CUDA_MOCK -eq 1 ]; then
+if [ $ENABLE_CUDA -eq 1 ] && [ $ENABLE_CUDA_MOCK -eq 1 ]; then
   show_error "You can't use both CUDA and CUDA mock plugins at once!"
   show_error "Please remove --enable-cuda or --enable-cuda-mock flag to continue."
   exit -1
@@ -660,7 +660,7 @@ check_snappy
 $PANTS export '::'
 
 if [ $ENABLE_CUDA_MOCK -eq 1 ]; then
-  cp "configs/accelerator/cuda-mock-enterprise.toml" cuda-mock.toml
+  cp "configs/accelerator/cuda-mock.toml" cuda-mock.toml
 fi
 
 # configure manager

